geth/transactions/txqueue_manager.go

```diff
@@ -83,6 +83,9 @@ func (m *Manager) TransactionQueue() *queue.TxQueue {
 
 // QueueTransaction puts a transaction into the queue.
 func (m *Manager) QueueTransaction(tx *common.QueuedTx) error {
+	if !tx.Args.Valid() {
+		return common.ErrInvalidSendTxArgs
```

Comment:
i would give more context to a user why we think that tx args are not valid

Reply:
That is a part of the error message (`are both 'input' and 'data' fields used?`): https://github.com/status-im/status-go/pull/758/files#diff-dfcab1c385b11aa286e849570d330006R28

Reply:
thanks, didnt notice that